mpv.rb

```diff
@@ -0,0 +1,68 @@
+class Mpv < Formula
+  # based upon the last version of homebrew-core/Formula/mpv.rb at https://github.com/Homebrew/homebrew-core/commit/41444d526c40b93069b7f0c5414539deb0534179
+  desc "Media player based on MPlayer and mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/privatezero/mpv/archive/2019-10-11.tar.gz"
+  version "2019-10-11"
+  sha256 "948eb483718cab755bbd081c9d1c43f101832801ad9db07390e4df4e409cde61"
+  head "https://github.com/mpv-player/mpv.git"
+
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python" => :build
+
+  depends_on "ffmpeg"
+  depends_on "jpeg"
+  depends_on "libarchive"
+  depends_on "libass"
+  depends_on "little-cms2"
+  depends_on "lua@5.1"
+
```

Comment:
This line can be deleted.

Reply:
The empty line is authentic to the history of this file, see https://github.com/Homebrew/homebrew-core/blob/03ac5ca999877ff8e522a354aaeb246815f715b5/Formula/mpv.rb#L24-L26

Reply:
Yet it can be fixed, in my opinion.

Reply:
There is some merit in minimizing the differences between this file and the one upstream, though I understand that sometimes a downstream fix may be more appropriate. Still I propose that it not block the PR, as the result of fixing the associated issue impacts how the installers and code behave rather than the cosmetics.

Reply:
It‘s not blocking at all.